### ISSUE TITLE
[BUG] Custom NBT Armor Stands & Spawn Eggs show as Out of Stock

### ISSUE THREAD

**Issue by SiuanSong** [snowgears/shopbugs#413]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Other unexpected behaviour

### Environment

```markdown
- Server Version: Purpur 1.20.1 #2050
- Shop Version: 1.8.4.4
```

### Server startup log

https://gist.github.com/SiuanSong/7049ee5b0a4520856ae67d829d17751a

### Shop config files

https://gist.github.com/SiuanSong/3381ab0d50d2f806fcbe5eedc026de23

### Bug description

Items with custom NBT data will eventually show as out of stock even when they aren't. Specific items this has happened with is posed armor stands and custom spawn eggs, both created through vanilla command block means.

### Steps to reproduce

1. Create any shop type
2. Sell or Barter a custom NBT data armor stand or spawn egg
3. Restart the server
4. Open the shop chest

### Expected behaviour

For the shop to remain in stock

### Actual behaviour

Shop sign displays as out of stock when it isn't

**Comment by OstlerDev** [comment]
Verified and Fixed in upcoming release 1.9.1.0

**Comment by OstlerDev** [comment]
Released 1.9.1.0
